Formula/qt.rb

```diff
@@ -23,134 +23,210 @@ class Qt < Formula
     sha256 cellar: :any, mojave:        "25c4a693c787860b090685ac5cbeea18128d4d6361eed5b1bfed1b16ff6e4494"
   end
 
-  keg_only "Qt 5 has CMake issues when linked"
-
+  depends_on "cmake" => [:build, :test]
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on xcode: :build
-  depends_on macos: :sierra
 
-  uses_from_macos "gperf" => :build
-  uses_from_macos "bison"
-  uses_from_macos "flex"
+  depends_on "assimp"
```

Comment:
If we're using system `assimp`, why is there still a `depends_on "assimp"` here?

Reply:
here means qt use system provided assimp(from homebrew)